ci/docker/runtime_functions.sh

```diff
@@ -77,7 +82,7 @@ build_armv6() {
         -DUSE_OPENCV=OFF \
         -DUSE_OPENMP=OFF \
         -DUSE_SIGNAL_HANDLER=ON \
-        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_BUILD_TYPE=RelWithDebInfo \
```

Comment:
I think this should stay Release, otherwise it's too big

Reply:
done